pom.xml

```diff
@@ -35,12 +32,12 @@
 		<repository>
 			<id>ms-releases</id>
 			<name>GitHub Release Repository</name>
```

Comment:
Technically it's the local release repository :). Also we could switch `id` from ms-narcissist to something like `local-releases` or something :)?